tests/virt/cluster/common_templates/utils.py

```diff
@@ -123,8 +123,8 @@ def _get_user_info(vm):
         cnv_info = get_cnv_user_info(vm=vm)
         libvirt_info = get_libvirt_user_info(vm=vm)
         linux_info = get_linux_user_info(ssh_exec=vm.ssh_exec)
-        if is_jira_63874_bug_open():
-            LOGGER.warning("Due to bug CNV-63874, loginTime is not available in virtctl output.")
+        if is_jira_64776_bug_open():
+            LOGGER.warning("Due to bug CNV-64776, loginTime is a bit big different between Libvirt level and OS level.")
             linux_info["loginTime"] = 0
```

Comment:
new bug need to be treated a bit different:
1. remove `linux_info["loginTime"] = 0`
2. add `virtctl_info["loginTime"] = int(virtctl_info["loginTime"] / 1000)`